README.md

```diff
@@ -19,9 +19,9 @@
 2. Create a webapp
```

Comment:
Should we remove the part "1. Run application" above? It seems outdated to me, or maybe I am missing something.

Reply:
Done